public/js/draw/setup.js

```diff
@@ -86,6 +86,19 @@ $('#finish-region').click(function () {
     finishRegion();
     });
 
+$('#submit-gid').click(function(){
+       $.ajax({
+            url: window.location.protocol + '//' + window.location.host + '/get-json-data?gid=' + $('#area-of-study').val(),
```

Comment:
The `url` can be relative, e.g. just `get-json-data`.

Pass `gid` as data for the request - see the `data` key at http://api.jquery.com/jquery.ajax/.